docs/extension/guides/set-wallet-recovery.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: '🦊 Set Wallet Recovery'
+sidebar_position: 6
+description: 'Extension Support: How to set up wallet recovery'
+---
+
+# Set Wallet Recovery
+
+If you want an autonomous backup for your Universal Profile, you can add an existing wallet as a recovery controller.
```

Comment:
cf other comment "autonomous backup" seems weird to me.

I would change "existing wallet" with "another wallet"

This wallet may not exist yet. The main idea is to set another wallet.